doc/overlapping.md

```diff
@@ -0,0 +1,145 @@
+|        X       |      X     |        |              |              |           X           |       X       |        X       | User Price        |
+|        X       |            |        |              |              |                       |       X       |                | External Price    |
+|        X       |            |        |              |              |                       |               |                | -                 |
+|        X       |      X     |    X   |              |              |                       |       X       |                | External Price    |
+|        X       |            |        |       X      |              |                       |       X       |                | Buy Marginal      |
+|        X       |            |        |       X      |              |           X           |       X       |                | External Price    |
+|        X       |            |        |       X      |              |           X           |               |                | Buy Marginal      |
+|        X       |            |        |       X      |              |           X           |               |        X       | User Price        |
+|        X       |            |        |              |       X      |                       |       X       |                | Sell Marginal     |
+|                |            |        |              |              |                       |       X       |                | External Price    |
+|                |            |        |              |              |                       |               |                | Calculated Price  |
+|                |            |    X   |              |              |                       |               |                | -                 |
+|                |            |        |       X      |              |                       |       X       |                | External Price    |
+|                |            |        |       X      |              |                       |               |                | Buy Marginal      |
+
+If there is not market price defined, we ask the user to set the **User Market Price**.
+
+## Calculate Order
+For **both Edit Price & Edit Budget** we'll calculate the new prices & budgets based on conditions
+
+### Calculate Price
+We need to calculate buy.max, buy.marginalPrice, sell.min & sell.marginalPrice.
+If strategy is **not touched** we use existing values, else we calculate them.
+
+To calculate prices we need these information : 
+- `marketPrice` (defined above)
+- `min` (default to `marketPrice * 0.99`)
+- `max` (default to `marketPrice * 1.01`)
+- `spread` (default to `0.05`)
+
+
+### Calculate Budget
+We need to calculate how the budget is impacted by a change in the strategy.
+The user will select an anchor token and provide the amount to deposit or withdraw
+
+Because of a precision error if **not touched** & user budget is zero: **do not calculate budget**.
+
```

Comment:
I would add an explanation here just to clarify what the +/- means:

```suggestion
Because of a precision error if **not touched** & user budget is zero: **do not calculate budget**. 

The user budget and calculated budget will be added or subtracted depending on whether the user wishes to deposit or withdraw.

```